lib/cictl/solr_command.rb

```diff
@@ -0,0 +1,48 @@
+require "thor"
+
+require_relative "common"
+require_relative "solr_client"
+
+module CICTL
+  class SolrCommand < Thor
+    include Common
+
+    desc "commit", "Send a commit command to solr"
+
+    def commit
+      solr_client.commit!
+    end
+
+    desc "dump_deleted FILE", "Dump the solr docs marked as `deleted` to the given .jsonl file"
```

Comment:
I think the least-surprising thing to do is use hyphenated commands `dump-deleted` and `send-jsonl`. Thor should map hyphenated `desc` to underscored `def`. And if you wanted to preserve the underscore option Thor's `map` directive should allow such synonyms but I haven't experimented with that.